### PR TITLE
refactor: Single Storybook instance instead of per-package

### PR DESCRIPTION
**Pull request by lukelooker**
This change simplifies our monorepo Storybook setup to use a single instance of Storybook for development and story-shots rather the the previous, per-package configuration.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "clean:storybook": "rm -Rf packages/*/storybook-static ",
     "clean:docs": "find docs -type f ! -name 'index.html' -delete",
     "clean:gatsby": "rm -Rf www/.cache www/public",
-    "clean:coverage": "rm -rf coverage",
```
Comment (mdodgelooker) [inline package.json:37]: This isn't needed anymore?

File: package.json [hunk 2]
```diff
@@ -48,10 +48,10 @@
     "test": "yarn jest",
 
     "storybook": "yarn workspace storybook develop",
-    "storybook-docs": "yarn workspace storybook develop-docs",
+    "storybook-nodocs": "yarn workspace storybook develop",
```
Comment (mdodgelooker) [inline package.json:51]: I think you probably meant to remove this
```suggestion
```

**Review by mdodgelooker — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (lukelooker) [reply package.json:37]: Whoops. Mistake

File: package.json (hunk 2 above)
Comment (lukelooker) [reply package.json]: Yep that's should go away 🤦

**Comment by lukelooker** [comment]
@mdodgelooker Fixed what you caught and went ahead and removed Storybook step from CI since this needed to re-run anyway.

**Review by mdodgelooker — APPROVED** [review]
👍

**Pull request merged**